recipes/dooit-extras/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+context:
+  name: dooit-extras
+  version: "0.2.0"
+  python_min: "3.9"
```

Comment:
You need that only if your min Python is higher than conda-forge's min. We do try to stay in sync with https://devguide.python.org/versions/  

```suggestion
```